ui/src/message_popup/experience_page.jsx

```diff
@@ -127,9 +136,16 @@ export default React.createClass({
               primaryText="Restart game" />
           }
         />
+        <Dialog
```

Comment:
We talked about removing this, but also as it is written now this would affect the simple "non-mobile prototype" interface too.  When making larger changes like this it's best if we can isolate the changes to only the new UI code, like within `renderMobilePrototype` or code it uses.